docs/settings.rst

```diff
@@ -141,6 +141,25 @@ Language Detection
     datetime.datetime(2015, 12, 4, 0, 0)
 
 
+Default Languages
++++++++++++++++++
+
+``DEFAULT_LANGUAGES``: It is a ``list`` of language codes in ISO 639 which is used as default 
+languages and are to be parsed. eg. "en":
```

Comment:
Setting this would essentially disable dateparser language detection if we don't pass any language and don't use custom language detection, right? I think we should clarify this behaviour in some place, because I think it's not obvious.

Reply:
`DEFAULT_LANGUAGES` is added after this language detection process.

I will add more information about it.

Reply:
done 👍🏻